zuul.d/edpm_multinode.yaml

```diff
@@ -0,0 +1,167 @@
+---
+# Base job definition for multinode. Provide base layout with CRC on a dedicated
+# nodeset, one compute and an ansible-controller.
+# At the end, you get an env ready to run the first stages of the ci-framework.
+# It also ensures post-run are common to all of the multinodes jobs we'll get
+# in that file.
+# In case you change the "nodeset" for your own job, please override
+# crc_ci_bootstrap_networking using *extra-vars*.
+- job:
+    name: cifmw-podified-edpm-base-multinode-crc
+    parent: base-extracted-crc
+    timeout: 10800
+    attempts: 1
+    nodeset:
+      nodes:
+        - name: controller
+          label: cloud-centos-9-stream-tripleo-vexxhost
+        - name: compute
+          label: cloud-centos-9-stream-tripleo-vexxhost
+        - name: crc
+          label: coreos-crc-extracted-xxl
```

Comment:
We keep all the nodeset definitions in this file https://github.com/openstack-k8s-operators/ci-framework/blob/main/zuul.d/nodeset.yaml . Can we move this nodeset definition to the same file?